search/collector/eligible.go

```diff
@@ -23,28 +23,40 @@ import (
 	index "github.com/blevesearch/bleve_index_api"
 )
 
+type eligibleStore struct {
+	ids []index.IndexInternalID
+}
+
+func (s *eligibleStore) AddDocumentMatch(doc *search.DocumentMatch) *search.DocumentMatch {
+	copyOfID := make([]byte, len(doc.IndexInternalID))
+	copy(copyOfID, doc.IndexInternalID)
+	s.ids = append(s.ids, copyOfID)
+	return doc
+}
+
 type EligibleCollector struct {
 	size    int
 	total   uint64
 	took    time.Duration
 	results search.DocumentMatchCollection
 
-	store collectorStore
+	store *eligibleStore
 }
 
 func NewEligibleCollector(size int) *EligibleCollector {
 	return newEligibleCollector(size)
 }
 
+func getEligibleCollectorStore(size int) *eligibleStore {
```

Comment:
Bad naming, and I don't really see the need for this method, use it's contents directly in line 59 below.